raiden/message_handler.py

```diff
@@ -113,6 +122,13 @@ def handle_message_withdrawrequest(
         raiden: "RaidenService", message: WithdrawRequest  # pylint: disable=unused-argument
     ) -> List[StateChange]:
         assert message.sender, "message must be signed"
+
+        sender_metadata: Optional[AddressMetadata] = None
+        if raiden.config.pfs_config is not None:
+            # HACK querying the PFS for the address-metadata directly after receiving a message
```

Comment:
```diff
-             # HACK querying the PFS for the address-metadata directly after receiving a message
+             # FIXME querying the PFS for the address-metadata directly after receiving a message
```
please change the marker to something we easily find in the future.
This is a tricky codepath as discussed before and we probably want to fix the behavior soon!